openstack/networking/v2/subnets/requests.go

```diff
@@ -79,7 +79,7 @@ type CreateOpts struct {
 	Name            string                `json:"name,omitempty"`
 	TenantID        string                `json:"tenant_id,omitempty"`
 	AllocationPools []AllocationPool      `json:"allocation_pools,omitempty"`
-	GatewayIP       *string               `json:"gateway_ip"`
+	GatewayIP       *string               `json:"-"`
```

Comment:
It's not working for me with `omitempty`. 

To confirm: the only change you made to master was the addition of `omitempty` to the `GatewayIP` tag?

Are you printing the map after `omitempty` would have stripped away the nil value? Isn't that the point of `omitempty`?

I'm printing `b` right after it was created in `Create` and either it's omitted (because it was nil) or it has a value of `""` (which Neutron does not want).